spec/controllers/content_items_controller_spec.rb

```diff
@@ -71,12 +78,20 @@
 
     it "assigns a list of organisations" do
       organisations = [build(:organisation), build(:organisation)]
-      allow(Organisation).to receive(:all).and_return(organisations)
+      allow(Organisation).to receive(:order).and_return(organisations)
       get :filter
 
       expect(assigns(:organisations)).to match_array(organisations)
     end
 
+    it "assigns a list of taxonomies" do
+      taxonomies = [build(:taxonomy), build(:taxonomy)]
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/95/commits/a1cf0a9d093b0e7a9ba237bebca59431468c6c1b